NameSorterAlpha/NameSorterAlpha/AlternateName.cs

```diff
@@ -27,24 +27,23 @@ public Person(string info, int line_num)
                 GivenNames += infoArray[i];
                 GivenNames += ' ';
             }
-            GivenNames.TrimEnd(new char[] {' '}); 
+            GivenNames.TrimEnd(new char[] {' '});
 
-            try
-            {
-                _dateOfBirth = DateTime.ParseExact(infoArray[length - 1], dateFormat, CultureInfo.InvariantCulture);
-            }
-            catch (Exception)
+            if (DateTime.TryParseExact(infoArray[length - 1], dateFormat, CultureInfo.InvariantCulture, DateTimeStyles.None, 
+                out _dateOfBirth)) { }
+
+            else
             {
-                Console.WriteLine("The person in line " + line_num + " is missing DOB");
+                Console.WriteLine("The person in line " + line_num + " is missing DOB.\n");
```

Comment:
If you're going to throw MissingInfoException earlier, this should also throw the same exception.